3rdParty/StormLib/src/FileStream.cpp

```diff
@@ -21,6 +21,11 @@
 #ifdef _MSC_VER
 #pragma comment(lib, "wininet.lib")             // Internet functions for HTTP stream
 #pragma warning(disable: 4800)                  // 'BOOL' : forcing value to bool 'true' or 'false' (performance warning)
+#else
+#include <cstring>

```

Comment:
Xadhoom: Edit StormPort.h cleanly to work on MacOS X correctly. As already in StormPort.h, use <string.h> instead of <cstring> etc (if there are no MacOS X quirks that require this). The includes in the separate source files should then not be necessary. And add the StormLib changes to the readme in the StormLib directory